src/Model/Buffer.re

```diff
@@ -15,6 +15,7 @@ type t = {
   lines: array(string),
   indentation: option(IndentationSettings.t),
   syntaxHighlightingEnabled: bool,
+  lastUsed: float,
```

Comment:
This ought to be modeled as an `option(float)`, but I think that's more bother than it's worth. If at some point we do need to know that a buffer has never been used, it'll be most likely be just as simple to change it then.